src/sentry/lang/native/plugin.py

```diff
@@ -155,7 +159,6 @@ def on_referenced(dsym_file):
         self.sym = Symbolizer(
             self.project,
             self.object_lookup,
-            arch=self.arch,
```

Comment:
The architecture is not needed here anymore. The object lookup will already resolve the correct architecture and symcaches are generated separately per architecture.